How many of the Technology Skills examples are in ERP, or CAD, or neither?

In [7]:
import pandas as pd
ts = pd.read_excel(r'Technology Skills.xlsx')
ts.rename(columns={"O*NET-SOC Code": "Onetcode",
    "Commodity Code" :  "CCode",
    "Commodity Title":  "CTitle",
    "Hot Techology":    "HotTech"}, inplace=True)

# hot techs only; where the column HotTech = "Y"
tsh = ts.loc[ (ts.iloc[:,5] == 'Y')]

In [8]:
df = tsh.copy()
df.rename(columns={
    "Commodity Code" :  "CCode",
    "Commodity Title":  "CTitle",
    "Hot Techology":    "HotTech"}, inplace=True)

In [9]:
# How many unique Examples among the whole table?
tshu = df.Example.unique()
tshu.shape

(805,)

In [10]:
# How many erp, unique?  Answer:  275 
erp = df.loc[ df.CCode == 43231602 ]
erpu = erp.Example.unique()
erpu.shape

(275,)

In [11]:
# How many CAD, unique?  Answer:  355 
cad = df.loc[ df.CCode == 43232604 ]
cadu = cad.Example.unique()
cadu.shape

(355,)

In [12]:
#How many neither CAD nor ERP, unique?  
neither = df.loc[ ~ df.CCode.isin([43231602,43232604])]
neitheru = neither.Example.unique()
neitheru.shape

(181,)

Expected 175, because 805 = 275+355+175; 
805 = All unique examples
275 = ERP
355 = CAD
181 = neither
Doesn't add up